<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Будильник</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>390</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>62</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>12:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTimeEdit" name="setTimeAlarm">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>405</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startAlarmBut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>195</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Завести будильник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editMusicBut">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>180</y>
      <width>195</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать мелодию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopAlarmBut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>390</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Остановить будильник</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
